dash/development/component_generator.py

```diff
@@ -98,8 +98,15 @@ def generate_components(
             os.makedirs('man')
         if not os.path.exists('R'):
             os.makedirs('R')
+        if os.path.isfile("dash-info.yaml"):
+            with open("dash-info.yaml") as yamldata:
+                rpkg_data = yaml.safe_load(yamldata)
```

Comment:
We need a fallback declaration of `rpkg_data` if `dash-info.yaml` doesn't exist - like we had [previously](https://github.com/plotly/dash/pull/835/files#diff-5704162f659bd0685c89d923fa806ed5L120):
```py
else:
    rpkg_data = None
```

(alternatively, initialize it to `None` up above the `if os.path.isfile("dash-info.yaml")` - just need to ensure it always exists by the time we ask for it below)

Reply:
Good catch. Sometimes I wish there was a ⚾️ mitt emoji.

Reply:
fixed in 139527d